### PR TITLE
Adding Dockerfile for BLAZE

### PR DESCRIPTION
**Pull request by atrull314**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [X] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [X] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [X] LABEL base_image  
 - [X] LABEL version
 - [X] LABEL software.version  
 - [X] LABEL about.summary  
 - [X] LABEL about.home  
 - [X] LABEL about.license   
 - [X] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [X] LABEL extra.identifier    
  - [ ] LABEL about.documentation   
  - [X] LABEL about.license_file
  - [ ] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

File: blaze/2.2.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+################## INSTALLATION ######################
+
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y \
+    build-essential \
+    libhdf5-dev
+
+RUN pip install --upgrade pip
+RUN pip install blaze2==2.2.0
```
Comment (mboudet) [inline blaze/2.2.0/Dockerfile]:
```suggestion
RUN pip install --no-cache-dir --upgrade pip \\
        && pip install --no-cache-dir blaze2==2.2.0
```
Can you merge both commands (and use --no-cache-dir ?)

File: blaze/2.2.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+################## MAINTAINER ######################
+MAINTAINER "Austyn Trull <atrull@uab.edu>"
+
+################## INSTALLATION ######################
+
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y \
+    build-essential \
+    libhdf5-dev
```
Comment (mboudet) [inline blaze/2.2.0/Dockerfile]:
```suggestion
RUN apt-get update && apt-get install --no-install-recommends -y \
        build-essential \
        libhdf5-dev && \
    apt-get clean && \
    apt-get purge && \
    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```
Do you need the apt upgrade?
(Also same thing, would be best to merge both commands, and clear the cache after)

File: blaze/2.2.0/Dockerfile (hunk 1 above)
Comment (atrull314) [reply blaze/2.2.0/Dockerfile]: Done in commit below. Thank you for the suggestion!

File: blaze/2.2.0/Dockerfile (hunk 2 above)
Comment (atrull314) [reply blaze/2.2.0/Dockerfile]: I was using another Dockerfile as base and it was present there, but it still works without it so I have removed it and merged as suggested. Thank you for the help!

**Pull request merged**